backend/tarsy/hooks/typed_history_hooks.py

```
@@ -42,11 +42,11 @@ async def execute(self, interaction: LLMInteraction) -> None:
             )
             if ok:
                 logger.debug(
-                    f"Stored LLM interaction {interaction.request_id} to history"
+                    f"Stored LLM interaction {interaction.interaction_id} to history"
                 )
             else:
                 logger.warning(
-                    f"History service returned False for LLM interaction {interaction.request_id}"
+                    f"History service returned False for LLM interaction {interaction.interaction_id}"
```

Ruff (0.12.2)

Line too long (102 > 88)

(E501)